packages/pnpm/test/install/hooks.ts

```diff
@@ -220,32 +220,32 @@ test('readPackage hook during update', async () => {
   await project.storeHas('dep-of-pkg-with-1-dep', '100.0.0')
 })
 
-test('prints meaningful error when there is syntax error in pnpmfile.js', async () => {
+test('prints meaningful error when there is syntax error in .pnpmfile.cjs', async () => {
   prepare()
 
-  await fs.writeFile('pnpmfile.js', '/boom', 'utf8')
+  await fs.writeFile('.pnpmfile.cjs', '/boom', 'utf8')
 
   const proc = execPnpmSync(['install', 'pkg-with-1-dep'])
 
   expect(proc.stderr.toString().includes('SyntaxError: Invalid regular expression: missing /')).toBeTruthy()
   expect(proc.status).toBe(1)
 })
 
-test('fails when pnpmfile.js requires a non-existend module', async () => {
+test('fails when .pnpmfile.cjs requires a non-existend module', async () => {
   prepare()
 
-  await fs.writeFile('pnpmfile.js', 'module.exports = require("./this-does-node-exist")', 'utf8')
+  await fs.writeFile('.pnpmfile.cjs', 'module.exports = require("./this-does-node-exist")', 'utf8')
 
   const proc = execPnpmSync(['install', 'pkg-with-1-dep'])
 
   expect(proc.stdout.toString().includes('Error during pnpmfile execution')).toBeTruthy()
```

Comment:
Using
```suggestion
  expect(proc.stdout.toString()).toContain('Error during pnpmfile execution')
```
will make the CI failure easier to debug.